src/cloudant/result.py

```diff
@@ -236,10 +236,47 @@ class Result(object):
     | Iteration                     | ``limit``, ``skip`` not permitted                         |
     +-------------------------------+-----------------------------------------------------------+
 
+    List of valid Result object customization options:
+    :param bool descending: Return documents in descending key order.
+    :param endkey: Stop returning records at this specified key.
+        Not valid when used with :class:`~cloudant.result.Result` key
```

Comment:
Since we are already in the Result class there is no need for a link to the ~cloudant.result.Result documentation.  I think it is enough to just say "Not valid when used with key access and key slicing.

You should apply this here and to lines 254, 257, 261, 265, and 271.